charts/matrix-stack/configs/haproxy/haproxy.cfg.tpl

```diff
@@ -82,15 +82,23 @@ resolvers kubedns
   hold refused 600s
 
 frontend prometheus
+{{- if has $root.Values.networking.ipFamily (list "ipv4" "dual-stack") }}
   bind *:8405
-  bind [::]:8405
+{{- end }}
+{{- if has $root.Values.networking.ipFamily (list "ipv6" "dual-stack") }}
+  bind [::]:8405 v6only
```

Comment:
Can you mention the comment from the PR here, I suspect it will be rather confusing for future selves.